src/styles/variables.scss

```diff
@@ -0,0 +1,18 @@
+$ds: 100px;
+$gp: 1px;
+$pd: 10px;
+$fs: 30px;
+$bs: border-box;
```

Comment:
ds, gp, pd, fs - I have no idea what variables are they. A variable name should describe it